corehq/apps/data_cleaning/templates/data_cleaning/status/complete.html

```diff
@@ -1,22 +1,22 @@
+{% extends "data_cleaning/status/base_modal_body.html" %}
 {% load i18n %}
 
-<div class="modal-header">
-  <h5 class="modal-title">
-    {% trans "Changes Applied" %}
-  </h5>
-</div>
-<div class="modal-body">
-  <div class="alert alert-success">
-    {% include "data_cleaning/status/partial/task_progress_bar.html" %}
-    <i class="fa-solid fa-check-double me-1"></i>
-    {# prettier-ignore-start #}
-    {% blocktrans count count=num_records_changed %}
-      Finished processing changes, updated 1 case.
-    {% plural %}
-      Finished processing changes, updated {{ num_records_changed }} cases.
-    {% endblocktrans %}
-    {# prettier-ignore-end #}
-  </div>
+{% block modal-header %}
+  {% trans "Changes Applied" %}
+{% endblock modal-header %}
+
+{% block task-status-text %}
+  <i class="fa-solid fa-check-double me-1"></i>
+  {# prettier-ignore-start #}
+  {% blocktrans count count=num_records_changed %}
+    Finished processing changes, updated 1 case.
+  {% plural %}
+    Finished processing changes, updated {{ num_records_changed }} cases.
```

Comment:
Fancy! I think I understand this, apart from `count` defined twice? But I'm assuming `count` is some special var that will factor into whether we use the singular or `{% plural %}` option?

Reply:
count is a special var for blocktrans, yes

Reply:
And why does it need to be both specified and then assigned as opposed to just assigned?